### ISSUE TITLE
[Feature request]: Use stylus to insert text

### ISSUE THREAD

**Issue by gieselbusch** [LinwoodDev/Butterfly#675]
### Describe your feature request!

On the Google Pixel Tablet you can use a pen or stylus to insert text in a pretty easy way. The recognition of the handwriting works great. This may be true for other tablets too.
It would be really nice to use feature inside butterfly. Unfortunately the pen could just be used to place a label but not to write text.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gieselbusch** [comment]
Hi,
I don't know how this exactly works. It might be, that the handwriting recognition is handled like a keyboard by the system. In most apps I can write anywhere on the screen and the app will take this as an input at the position of the cursor.
It's shortly shown in this [video (yt)](https://youtu.be/AwrOWrlhESI?feature=shared&t=306). But that's not the exact feature, since he has to write in a specific area.

**Comment by MrGumpIT** [comment]
I think that's what you mean

Uploading screenrecorder-20240513-224854~2.mp4…

**Comment by gieselbusch** [comment]
Yes, exactly. But in most apps you even don't have to write in the URL field, which makes the usage a lot more comfortably.

**Comment by pacjo** [comment]
I believe gboard calls it [write in text  fields](https://9to5google.com/2023/12/17/gboard-stylus-handwriting/) and it has something to do with:

![Screenshot_20240803-225113_Settings](https://github.com/user-attachments/assets/77c3948c-2fc3-4217-b764-8ca57e2b857d)

Also there's another way of implementing this, where you keep handwritten text, but you save it's recognized form somewhere, so it becomes indexable. (but that's probably a different is issue)

and with a full (I don't know flutter, so I don't know if it's relevant) disclaimer out at the way, this package looks interesting: https://pub.dev/packages/google_mlkit_digital_ink_recognition

**Comment by CodeDoctorDE** [comment]
Hmm, the thing is: This is not native. There you need to download an ai model to use it.
I don't know if it is worth the time. If gboard can do it, this wouold solve the issue here i guess?

**Comment by pacjo** [comment]
> This is not native.

yeah...

> There you need to download an ai model to use it.

really? I must have missed that part. That's probably a licensing issue too?

> If gboard can do it, this wouold solve the issue here i guess?

Possibly, but I think that's for the author to decide, @gieselbusch? 

In case it's closed, should I open another issue for the indexing suggestion?

**Comment by CodeDoctorDE** [comment]
Looks like it was added to flutter recently: https://github.com/flutter/flutter/pull/148784.
Now we need to wait until it's in the stable branch.
Then we can see how we can implement it inside the label tool. Sadly I don't have such a device that has a stylus

**Comment by CodeDoctorDE** [comment]
Should be available in 2.4.0-beta.0 in the next weeks
